test/server-test.js

```diff
@@ -1,13 +1,30 @@
 import { assert } from 'chai';
-
+import express from 'express';
 import hypernova from '../server';
 
 describe('Hypernova server', () => {
+  it('blows up if hypernova does not get a real express function', () => {
+    assert.throws(
+      hypernova.bind({ devMode: true, getComponent: () => {}, expressInstance: () => {} }),
+      TypeError);
+  });
+
   it('blows up if hypernova does not get getComponent', () => {
     assert.throws(hypernova, TypeError);
   });
 
-  it('starts up the hypernova server without blowing up', () => {
-    hypernova({ devMode: true, getComponent: () => {} });
+  // it('starts up the hypernova server without blowing up', () => {
```

Comment:
We had to comment this test because the logger library deletes the `init` method after it is initialized and it was breaking the other test. If you could help us or give us some guidance, that would be awesome! thanks

Reply:
<img width="791" alt="screen shot 2018-07-17 at 15 41 47" src="https://user-images.githubusercontent.com/33033260/42830514-92789076-89e3-11e8-9ab3-ea8c87ae0a76.png">

Reply:
In that case, whatever state is being persisted across tests needs to be reset before each test - `delete require.cache[require.resolve(‘logger’)]` or similar.